docs/registering-ids.md

```diff
@@ -3,38 +3,35 @@ id: registering-ids
 title: Registering Samples
 ---
 
-## What it means
+In the ARGO Data Platform, clinical and molecular data objects are assigned ARGO Identifiers (**ARGO IDs**) used to track the data through the Platform. Each **Donor**, **Specimen**, and **Sample** entity will be assigned an **ARGO ID**, mapped to your program's internal identifiers.
 
->Registration is the first step in the data submission life cycle - we mandate that it be done ahead of any clinical or molecular data submissions.
+It is important that the relationships between entities are  maintained across submissions, as they are fundamental to data tracking and integrity across the ARGO Platform.
 
-Clinical and molecular data is assigned to simple, and easy-to-read identifiers on the ARGO platform. Each **Donor**, **Specimen** & **Sample** entity will be assigned a respective **ARGO ID**, mapping to your program's internal identifiers. It is important that the relationships between these entities are  maintained across submissions, as they are fundamental for data processing and synchronization across all ARGO resources.
-
-In order to prevent data integrity errors in future clinical or molecular data submissions, we ask that you first register **Sample** data which includes :
+> Registration is the first step in the data submission life cycle.  You must registers samples before any clinical or molecular data submissions.
 
+The basic set of data that must be registered for each sample consists of:
 * Submitter Donor ID
 * Donor Gender
 * Submitter Specimen ID
+* Specimen Tissue Source
 * Specimen Tumor or Normal Designation
-* Specimen Type
 * Submitter Sample ID
 * Sample Type
 
+During sample registration, **ARGO IDs** will be assigned to your programs data. Any attempts to submit data  which does not refer to a previously registered samples will result in an error, and will prompt you to complete the registration step before proceeding.
```

Comment:
...to your program data... (remove the s)
Any attempts to submit clinical or molecular data that does not refer to previously registered samples will result in an error, i.e. you will be prompted to complete sample registration before proceeding.